### PR TITLE
Mobile optimisation

### PR DESCRIPTION
**Pull request by jb3**
New optimisations on mobile:
- Terminal removed in favor of error message 
image
- New dropdown on small screens
image
- Button fixed on mobile displays
image

### CODE REVIEW

File: static/css/window.css [hunk 1]
```diff
@@ -263,4 +263,10 @@ pre {
 
 .uk-background-muted, .error-header {
     background-color: #e8e8e8 !important;
+}
+
+code, pre {
+    width: 100%;
+    word-wrap: break-word;
+    overflow-wrap: break-word;
 }
```
Comment (gdude2002) [inline static/css/window.css]: Missing newline at end of file

File: templates/errors/error.html [hunk 2]
```diff
@@ -9,12 +9,15 @@
         window._Path = "{{ request.path }}"
     </script>
     <div class="uk-background-muted">
-        <div class="uk-section">
-            <div class="uk-text-center"><code class="error-header uk-text-large">{{ error_title }}</code></div>
+    <div class="uk-container">
+        <div class="uk-section uk-text-center uk-panel uk-panel-box">
+           <p style="color: #e84149; font-family: monospace;" class="error-header uk-text-large uk-text-break">{{ error_title }}</p>
```
Comment (gdude2002) [inline templates/errors/error.html]: Is there no UIKit class for this?

**Review by gdude2002 — CHANGES_REQUESTED** [review]
Does this still look the same on desktop?

**Review by lemonsaurus — DISMISSED** [review]
Apart from what gdude already mentioned, this looks production ready to me.

File: templates/errors/error.html (hunk 2 above)
Comment (gdude2002) [reply templates/errors/error.html]: There is no UIKit class for this

**Review by gdude2002 — APPROVED** [review]

**Review by lemonsaurus — APPROVED** [review]
excellent changes.

**Pull request merged**